src/content/development/contribution-guide/_index.en.md

```diff
@@ -336,7 +313,6 @@ Should a bug be identified during a test day, it should be labeled with an appro
 [test day template]: https://docs.google.com/spreadsheets/d/14wNJAbR2CC3wWdHtxnvnNlft8kSWp-SIFMWJ-hRISao
 [Submariner project]: https://github.com/submariner-io
 [triages tasks and bugs as part of a weekly meeting]: https://docs.google.com/document/d/1ICvOBOUpIEI45Yqcjf26Z1e5mZNzYsEN-H7_3xx54jI
-[Project Members]: https://submariner.io/community/contributor-roles/#member
```

Comment:
Why this change? Doesn't seem related.

Reply:
The link was referenced from a line which was removed in this PR.
Line 294, as part of the "Exception Process"